src/main/java/com/google/api/codegen/transformer/py/PythonGapicSurfaceTransformer.java

```diff
@@ -176,75 +187,143 @@ private void addFieldsImports(ModelTypeTable typeTable, Iterable<FieldModel> fie
     }
   }
 
-  private ViewModel generateApiClass(GapicInterfaceContext context) {
+  private List<ViewModel> generateApiAndSampleClasses(GapicInterfaceContext context) {
+    ImmutableList.Builder<ViewModel> viewModels = new ImmutableList.Builder<>();
     SurfaceNamer namer = context.getNamer();
     String subPath =
         pathMapper.getOutputPath(
             context.getInterfaceModel().getFullName(), context.getProductConfig());
     String name = namer.getApiWrapperClassName(context.getInterfaceConfig());
-    List<ApiMethodView> methods = generateApiMethods(context);
+    List<ApiAndSamples<OptionalArrayMethodView, OptionalArrayMethodView>> methods =
+        generateApiMethodsAndSamples(context);
 
-    DynamicLangXApiView.Builder xapiClass = DynamicLangXApiView.newBuilder();
-    xapiClass.templateFileName(XAPI_TEMPLATE_FILENAME);
-    xapiClass.outputPath(namer.getSourceFilePath(subPath, name));
+    DynamicLangXApiView.Builder xapiClassBuilder = DynamicLangXApiView.newBuilder();
```

Comment:
I'd prefer not to add `Builder` on the variables all over the place because it adds so much text.